build/configs/rtl8730e/loadable_apps/defconfig

```diff
@@ -1228,13 +1258,11 @@ CONFIG_BINFMT_ENABLE=y
 CONFIG_BINFMT_LOADABLE=y
 CONFIG_ELF=y
 CONFIG_ELF_ALIGN_LOG2=2
-CONFIG_ELF_STACKSIZE=2048
+CONFIG_ELF_STACKSIZE=8192
 CONFIG_ELF_BUFFERSIZE=32
 CONFIG_ELF_BUFFERINCR=32
 CONFIG_ELF_EXCLUDE_SYMBOLS=y
-CONFIG_ELF_CACHE_READ=y
-CONFIG_ELF_CACHE_BLOCK_SIZE=2048
-CONFIG_ELF_CACHE_BLOCKS_COUNT=60
+# CONFIG_ELF_CACHE_READ is not set
```

Comment:
Do you want to disable this? As I remember, your team enabled.
Could you confirm this with your team?

Reply:
Confirmed, should be enabled